dockyard/api/config.py

```diff
@@ -0,0 +1,24 @@
+# Pecan Application Configurations
+app = {
+    'root': 'dockyard.controllers.root.RootController',
+    'modules': ['dockyard', 'dockyard.api'],
```

Comment:
As per documentation, for modules key values should be one. You can have two values but recommended is one.
It's should be in the same directory, where your app.py file is located.

Therefor correct it and put only one required value.